test/test.js

```diff
@@ -421,15 +427,48 @@ ruleTester.run('generated-flow-types', rules['generated-flow-types'], {
   invalid: [
     {
       code: `
-        import type {TestFragment_other} from './path/to/TestFragment_other.graphql';
+        import type {TestFragment_other$ref} from './path/to/TestFragment_other.graphql';
+        useFragment<FooResponse>(graphql\`query TestFragment_foo { id }\`)
+      `,
+      errors: [
+        {
+          message: `
+The prop passed to useFragment() should be typed with the type 'TestFragment_foo$ref' imported from 'TestFragment_foo.graphql', e.g.:
+
+  import type {TestFragment_foo} from 'TestFragment_foo.graphql';`.trim(),
+          line: 3,
+          column: 9
+        }
+      ]
+    },
+    {
+      // Should import the type
```

Comment:
nit: for clarity, since it wasn't immediately obvious to me why this would be wrong

// Should import the type using the `type` keyword